Files/ViewModels/SettingsViewModels/AppearanceViewModel.cs

```diff
@@ -139,5 +142,26 @@ public bool AreRightClickContentMenuAnimationsEnabled
                 }
             }
         }
+        public string SelectedThemeName
+        {
+            get
+            {
+                return selectedThemeName;
+            }
+            set
+            {
+                if (SetProperty(ref selectedThemeName, value))
+                {
+                    App.AppSettings.PathToThemeFile = selectedThemeName;
+                    ShowRestartDialog = true;
+                }
+            }
+        }
+
+        public bool ShowRestartDialog
+        {
+            get => showRestartDialog;
+            set => SetProperty(ref showRestartDialog, value);
+        }
```

Comment:
Is it posible to swap out the themes at run time?

Reply:
@yaichenbaum I wasn't able to get it working, the appearance wouldn't change. I think the dictionary has to be added before the controls are rendered.